src/main/java/org/jetbrains/bsp/bazel/server/bazel/data/SemanticVersion.java

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.bsp.bazel.server.bazel.data;
+
+public class SemanticVersion implements Comparable<SemanticVersion> {
+
+  private final String version;
+
+  public final String get() {
+    return this.version;
+  }
+
+  public SemanticVersion(String version) {
+    if (version == null) throw new IllegalArgumentException("Version can not be null");
+    if (!version.matches("[0-9]+(\\.[0-9]+)*"))
+      throw new IllegalArgumentException("Invalid version format");
+    this.version = version;
+  }
+
+  static SemanticVersion fromReleaseData(String version) {
+    return new SemanticVersion(version.split(" ")[1]);
+  }
+
+  @Override
+  public int compareTo(SemanticVersion other) {
```

Comment:
As i understand versioning format correctly - it is `xx.xx`, right? Maybe we should explicitly extract first part and second (maybe, getMajorVersion, getMinorVersion, we can even keep it in this format) and then in comparison explicitly compare majors first and then minors

Reply:
Semantic versioning usually divides into three versions in a `xx.xx.xx` format. But I can do that either way!